manifests/charts/fleet-manager/templates/rbac.yaml

```diff
@@ -13,6 +13,49 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - apps.kurator.dev
+    resources:
+      - '*'
+    verbs:
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - source.toolkit.fluxcd.io
+    resources:
+      - '*'
+    verbs:
+      - create
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - kustomize.toolkit.fluxcd.io
+    resources:
+      - '*'
+    verbs:
+      - create
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - helm.toolkit.fluxcd.io
+    resources:
+      - '*'
+    verbs:
+      - create
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Comment:
IIRC, some of these were added in prev PR, can you kindly double check?

Reply:
source.toolkit.fluxcd.io and  helm.toolkit.fluxcd.io are duplicate,  it is unnecessary